### PR TITLE
[NUI] Fix timePicker ampm internal logic

### PR DESCRIPTION
**Pull request by lwc0917**
### Description of Change ###
https://github.sec.samsung.net/dotnet/NUIBackend/issues/108
Fix ampm time calc logic.

### API Changes ###
No API Changes

### CODE REVIEW

File: src/Tizen.NUI.Components/Controls/Picker.cs [hunk 1]
```diff
@@ -520,7 +520,10 @@ private void UpdateCurrentValue()
 
             if (loopEnabled) startY = ((dummyItemsForLoop + startItemIdx) * itemHeight) + startScrollOffset;
             // + 2 for non loop picker center align
-            else startY = ((2 + startItemIdx) * itemHeight) + startScrollOffset;
+            else {
```
Comment (dongsug-song) [inline src/Tizen.NUI.Components/Controls/Picker.cs:523]: just small thing.
could you follow the indentation rule?
it is better to 
`else` 
`{`
thank you.

**Review by jaehyun0cho — APPROVED** [review]
LGTM

File: src/Tizen.NUI.Components/Controls/Picker.cs (hunk 1 above)
Comment (lwc0917) [reply src/Tizen.NUI.Components/Controls/Picker.cs]: Thank you. i updated PR.

**Review by dongsug-song — APPROVED** [review]
looks good to me

**Pull request merged**